lib/msf/core/exploit/cmdstager.rb

```diff
@@ -24,7 +24,8 @@ module Exploit::CmdStager
     :tftp => Rex::Exploitation::CmdStagerTFTP,
     :wget => Rex::Exploitation::CmdStagerWget,
     :curl => Rex::Exploitation::CmdStagerCurl,
-    :fetch => Rex::Exploitation::CmdStagerFetch
+    :fetch => Rex::Exploitation::CmdStagerFetch,
+    :lwprequest => Rex::Exploitation::CmdStagerLwpRequest
```

Comment:
Can we rename this to `:lwp_request`?

Reply:
I understand it's `lwp-request`, so hmm. If you drop that char, there's no need to waffle.

Reply:
I am breaking new ground by introducing a utility whose name contains never before seen elements of ASCII.